notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-818_SITCOMTN-083.ipynb

```diff
@@ -4,8 +4,10 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    id = 108
</pre><p>Question: it is an id for an actuator which we wanna check, right? </p><p>If it is maybe you'd better add a new cell and put comment like </p><p>"put the number of actuator to see the result from bump test"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>